sncosmo/builtins.py

```diff
@@ -733,6 +733,62 @@ def load_snemo(relpath, name=None, version=None):
                              args=['models/snemo/'+file],
                              version=ver, meta=meta)
 
+
+# P18
+p18Models_CC = [('snana-2004fe', 'SN Ic', 'CSP-2004fe.SED'),
```

Comment:
These look like all the same names as the SNANA CC models (e.g. L628). How do they relate to the SNANA versions? Do you want to change the prefix of the model names here? (`snana-2004fe` -> `<prefix>-2004fe`)

Reply:
I can, up to you how you want to organize the models. These are all extensions of the existing SNANA models, with the ugriz bands untouched, the y band updated for the PLASTICC competition, and the JHK bands added based on the paper. I can keep them separate, but since the optical templates are really the same it feels a little like duplicating the repositories. Maybe we should just combine them somehow?

Reply:
Ah, I see. Right now there are errors due to trying to register a model with the same name and version as an existing one, so we need to either change the name or the version. Sounds like it might make sense to call these version 2.0 of the existing models, as they should be preferred in all cases. Does that make sense?

The only issue I foresee would be if someone was doing `sncosmo.Model(source='snana-2004fe')`, when they update sncosmo to a new version, they'll suddenly get wider wavelength coverage in that model. That might change their results or potentially cause an error. (To get the original model back, one can do `sncosmo.Model(source=sncosmo.get_source('snana-2004fe', version='1.0'))`). This was actually intended in the original design: you get the latest version by default and if you want a specific version, you specify it. But now I'm wondering how disruptive it would be to existing code. Any thoughts?

Reply:
I see the concern with existing code. In most cases it won't be a problem because users won't have been accessing the near-IR wavelengths with their code, since they didn't exist. But I can see there could be an issue if, for example, someone specifically did something like set a bound as wave[-1], expecting it to be in the z or y band, and suddenly it shifts out to the K band. 

I'd say either we just make these version 2.0 and make sure the documentation shows that these are changing the wavelength range and some of the y band data, or maybe follow what you did for SALT2 and give every one of these a name like "snana-2004fe-extended". A little more cumbersome but safer. Would you rather I do that?

For one thing though, the SALT2-extended model that I've uploaded should be indeed version 2.0 of the existing one, as it is an updated version of Steve's old one that is in there at the moment.

Reply:
Let's go with `version='2.0'` for all of them. If there is some major problem, we can always change it.